console/pom.xml

```diff
@@ -29,8 +29,9 @@
     <description>console for Seata built with Maven</description>
 
     <properties>
-        <spring-boot-for-server.version>${spring-boot.version}</spring-boot-for-server.version>
-        <spring-framework-for-server.version>${spring-framework.version}</spring-framework-for-server.version>
+        <spring-boot-for-server.version>2.7.17</spring-boot-for-server.version>
```

Comment:
Why modify here instead of directly modifying the main pom?

Reply:
The client SDK should try to use lower versions as much as possible to have more users. The server, as a separately deployed process, should try to avoid or minimize security vulnerabilities, which is decoupled from the user program.

Reply:
> Why modify here instead of directly modifying the main pom?

@slievrly is right.